src/main/java/com/stripe/model/Account.java

```diff
@@ -303,6 +303,18 @@ public Account update(Map<String, Object> params, RequestOptions options)
 		return request(RequestMethod.POST, instanceURL(Account.class, this.id), params, Account.class, options);
 	}
 
+	public DeletedAccount delete()
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return delete((RequestOptions) null);
+	}
+
+	public DeletedAccount delete(RequestOptions options)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return request(RequestMethod.DELETE, instanceURL(Account.class, this.id), null, DeletedAccount.class, options);
```

Comment:
IMO they should both call the other methods. They shouldn't make the request themselves so that there's only one path to a call to `request`